src/main/java/backend/Logic.java

```diff
@@ -271,6 +271,25 @@ public Model getRepo(String repoId) {
     }
 
     /**
+     * Replaces existing milestone with the newMilestone in the issue object, the UI, and the server, in that order.
+     * Server update is done after the local update to reduce the lag between the user action and the UI response
+     *
+     * @param issue The issue object containing the repoid and issue id of the relevant issue
+     *              whose milestone is to be replaced
```

Comment:
This part can be omitted? `containing the repoid and issue id of the relevant issue`